.circleci/config.yml

```diff
@@ -1,18 +1,13 @@
 version: 2.1
 
 workflows:
-  merge:
+  commit:
     jobs:
       - build:
-        filters:
-          branches:
-            only:
-              - main
-      - cache:
-        filters:
-          branches:
-            only:
-              - main
+          filters:
+            branches:
+              only:
+                - main
```

Comment:
The error is caused by the wrong indentation.. That's why I don't like a configuration syntax that is based on indentation..